dist/assets/d4d85ae49e726dcf637e2eacb3651afc.md

```diff
@@ -0,0 +1,212 @@
+# DC Candidate Fundraising Integrity Scorecard
```

Comment:
I think the long term goal is to create a component for it, but I think for now we should keep it as dangerouslySetInnerHtml